python/Ganga/Utility/execute.py

```diff
@@ -283,7 +283,7 @@ def execute(command,
         if stdout:
             stdout = pickle.loads(stdout)
     except Exception as err:
-        logger.error("Err: %s" % str(err))
+        logger.debug("Err: %s", err)
```

Comment:
Ahh, I think this used to be `debug` but you changed it to `error` recently and I rebased incorrectly. I will change it bask to `error`.